src/App.tsx

```diff
@@ -10,33 +11,19 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTabId] = useState(tabs[0]);
```

Comment:
So, as I said it's not an id
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```

Reply:
sorry my mistake